### PR TITLE
[Tuning] Diverse Windows Rules Tuning 

### PR DESCRIPTION
**Pull request by Samirbous**
Adjusting some windows rules.

### CODE REVIEW

File: rules/windows/defense_evasion_persistence_account_tokenfilterpolicy.toml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+]
+risk_score = 47
+rule_id = "07b1ef73-1fde-4a49-a34a-5dd40011b076"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion", "Privilege Escalation"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+registry where registry.path : "HKLM\\*\\LocalAccountTokenFilterPolicy" and registry.data.strings : ("1", "0x00000001")
```
Comment (w0rk3r) [inline rules/windows/defense_evasion_persistence_account_tokenfilterpolicy.toml:34]:
```suggestion
registry where registry.path : (
  "HKLM\\*\\LocalAccountTokenFilterPolicy",
  "\\REGISTRY\\MACHINE\\*\\LocalAccountTokenFilterPolicy") and
  registry.data.strings : ("1", "0x00000001")
```

Comment (w0rk3r) [reply rules/windows/defense_evasion_persistence_account_tokenfilterpolicy.toml:34]: To be Endgame compatible

File: rules/windows/defense_evasion_persistence_account_tokenfilterpolicy.toml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+name = "Local Account TokenFilter Policy Registry Modification"
+references = [
+    "https://www.stigviewer.com/stig/windows_server_2008_r2_member_server/2014-04-02/finding/V-36439",
+    "https://posts.specterops.io/pass-the-hash-is-dead-long-live-localaccounttokenfilterpolicy-506c25a7c167",
+    "https://www.welivesecurity.com/wp-content/uploads/2018/01/ESET_Turla_Mosquito.pdf"
+]
+risk_score = 47
+rule_id = "07b1ef73-1fde-4a49-a34a-5dd40011b076"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion", "Privilege Escalation"]
```
Comment (w0rk3r) [inline rules/windows/defense_evasion_persistence_account_tokenfilterpolicy.toml]:
```suggestion
tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion", "Privilege Escalation", "Elastic Endgame"]
```

File: rules/windows/credential_access_wireless_creds_dumping.toml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+creation_date = "2022/11/01"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/11/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempts to dump Wireless saved access keys in clear text using the Windows builtin utility Netsh.
```
Comment (w0rk3r) [inline rules/windows/credential_access_wireless_creds_dumping.toml:11]:
```suggestion
Identifies attempts to dump Wireless saved access keys in clear text using the Windows built-in utility Netsh.
```

File: rules/windows/discovery_files_dir_systeminfo_via_cmd.toml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+    [[rule.threat.technique.subtechnique]]
+    id = "T1059.003"
+    name = "Windows Command Shell"
+    reference = "https://attack.mitre.org/techniques/T1059/003/"
+
+[rule.threat.tactic]
+id = "TA0002"
+name = "Execution"
+reference = "https://attack.mitre.org/tactics/TA0002/"
```
Comment (w0rk3r) [inline rules/windows/discovery_files_dir_systeminfo_via_cmd.toml:71]:
```suggestion
reference = "https://attack.mitre.org/tactics/TA0002/"

```

File: rules/windows/persistence_user_account_creation_event_logs.toml [hunk 5]
```diff
@@ -17,18 +19,18 @@ false_positives = [
...
 license = "Elastic License v2"
-name = "Creation of a local user account"
+name = "Windows User Account Creation"
 risk_score = 21
 rule_id = "38e17753-f581-4644-84da-0d60a8318694"
 severity = "low"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
```
Comment (w0rk3r) [inline rules/windows/persistence_user_account_creation_event_logs.toml:30]: We may want to maintain it as KQL as we don't support converting types AFAIK

Comment (Samirbous) [reply rules/windows/persistence_user_account_creation_event_logs.toml]: switched back to kql, while testing rule didn't trigger, had to add `event.module : "system"` probably any rule using event.module in the kql query and `logs-system*` in index should be tested.

File: rules/windows/defense_evasion_persistence_account_tokenfilterpolicy.toml [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+min_stack_version = "8.3.0"
+updated_date = "2022/11/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies registry modification of the LocalAccountTokenFilterPolicy value. A compromised local administrator account
+can provide means for an attacker to move laterally between domain systems. With User Account Control enabled, filtering
+the privileged token for local administrator accounts will prevent the elevated privileges of these accounts from being
+used over the network.
```
Comment (w0rk3r) [inline rules/windows/defense_evasion_persistence_account_tokenfilterpolicy.toml:14]: I think we can reword the second part to describe how it works first and why an attacker may want to disable it after

**Review by w0rk3r — COMMENTED** [review]
Fire, as always

File: rules/windows/persistence_user_account_creation_event_logs.toml [hunk 7]
```diff
@@ -28,7 +28,7 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-event.module:security and event.code:4720
+event.module : ("system" or "security") and event.code:"4720"
```
Comment (w0rk3r) [inline rules/windows/persistence_user_account_creation_event_logs.toml:31]: Maybe we can use that event.action here

File: rules/windows/defense_evasion_persistence_account_tokenfilterpolicy.toml (hunk 6 above)
Comment (Samirbous) [reply rules/windows/defense_evasion_persistence_account_tokenfilterpolicy.toml]: reworded to be more specific.

**Review by w0rk3r — APPROVED** [review]
🧙🏼

**Review by DefSecSentinel — APPROVED** [review]
LGTM

**Pull request merged**